src/sentry/static/sentry/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -1,18 +1,26 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import styled from '@emotion/styled';
 
 import {openAddDashboardWidgetModal} from 'app/actionCreators/modal';
 import {loadOrganizationTags} from 'app/actionCreators/tags';
 import {Client} from 'app/api';
+import {
+  setBodyUserSelect,
+  UserSelectValues,
+} from 'app/components/events/interfaces/spans/utils';
```

Comment:
Should we move these to somewhere more general? These functions are used in a few different places now.

Reply:
I'll see if I can move them.